archetypal/idfclass/outputs.py

```diff
@@ -32,7 +32,7 @@ class Outputs:
         "Zone Lights Total Heating Energy",
     )
     ELECTRIC_EQUIP = (
-        "Zone Electric Equipment Electricity Energy",
+        "Zone Electric Equipment Electric Energy",
```

Comment:
This is an interesting one. Starting with E+ 9.4 (and above) the Name is "Zone Electric Equipment **Electricity** Energy". we should make a more robust way of specifying names that change based on the version.

Reply:
One solution could be to replace the line with something like this. Here EnergyPlusVersion.current() returns a Version object for the version specified in the archetypal module or the environment variable `ENERGYPLUS_VERSION`

```python

ELECTRIC_EQUIP = (
        "Zone Electric Equipment Electricity Energy" if EnergyPlusVersion.current() >= EnergyPlusVersion("9.4") else "Zone Electric Equipment Electric Energy",
        "Zone Electric Equipment Total Heating Energy",
        "Zone Electric Equipment Radiant Heating Energy",
        "Zone Electric Equipment Convective Heating Energy",
        "Zone Electric Equipment Latent Gain Energy",
    )
)
```

The problem with this is that since it is declared at the class level, if a user specifies a different version number for their IDF object - IDF(..., as_version="22-10") then there is no way of knowing that.

@szvsw any thoughts?

Reply:
Reverting to `Zone Electric Equipment Electricity Energy` for now